### ISSUE TITLE
pactus-wallet history add panic

### ISSUE THREAD

**Issue by Ja7ad** [pactus-project/pactus#1140]
### What Happened

when run command `pactus-wallet history add` get panic.

```shell
$ pactus-wallet history add

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.buildAddToHistoryCmd.func1(0xc0004e3400?, {0xc0004da640?, 0x4?, 0x105e9bc?})
	github.com/pactus-project/pactus/cmd/wallet/history.go:32 +0x138
github.com/spf13/cobra.(*Command).execute(0xc0004eac00, {0xc0004da600, 0x4, 0x4})
	github.com/spf13/cobra@v1.7.0/command.go:944 +0x863
github.com/spf13/cobra.(*Command).ExecuteC(0xc00037cc00)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
main.main()
	github.com/pactus-project/pactus/cmd/wallet/main.go:69 +0x21b

```

**Comment by kehiy** [comment]
Hi @Ja7ad, check the help flag or related source code you need to provide transaction id also.

That's not a bug itself. But we have to add a check to make sure there is enough arguments provided.

**Comment by kehiy** [comment]
@Ja7ad fixed in #1141
